git_machete/cmd.py

```diff
@@ -32,17 +32,31 @@ def __str__(self):
 RED = '\033[91m'
 
 
+ascii_only = False
+
+
 def bold(s):
-    return BOLD + s + ENDC
+    return BOLD + s + ENDC if not ascii_only else s
 
 
 def dim(s):
-    return DIM + s + ENDC
+    return DIM + s + ENDC if not ascii_only else s
 
 
 def underline(s):
-    return UNDERLINE + s + ENDC
+    return UNDERLINE + s + ENDC if not ascii_only else s + " *"
+
+
+def colored(s, color):
```

Comment:
I think there's still at least one place where a color (`YELLOW` specifically) is used in the old concatenated manner (hook executability check)... pls do a `grep` or whatnot across the file for other possible instances